tests/task/traversal_data.py

```diff
@@ -241,7 +242,8 @@ def sample_traversal() -> Traversal:
         name="mysql", collections=[customers, addresses, users], connection_key="mysql"
     )
     postgres = Dataset(name="postgres", collections=[orders], connection_key="postgres")
+    mssql = Dataset(name="mssql", collections=[addresses], connection_key="mssql")
 
-    graph = DatasetGraph(mysql, postgres)
+    graph = DatasetGraph(mysql, postgres, mssql)
```

Comment:
Do we need a follow-up to add `mongodb`, `redshift`, and `snowflake` here too? It looks like this has been overlooked

Reply:
We already have `mongo` traversal tests (this specific line is associated with SQL traversals). But I don't see any reason we can't add `redshift` and `snowflake`, too. Unless there's something specific to external connectors that makes it more difficult to test?